packages/mds-types/index.ts

```diff
@@ -262,6 +334,23 @@ export const TAXI_STATE_EVENT_MAP = TaxiStatusEventMap({
   stopped: ['trip_stop', 'reservation_stop']
 })
 
+export const TNC_STATE_EVENT_MAP = TncStatusEventMap({
+  available: [
+    'driver_cancellation',
+    'enter_jurisdiction',
+    'passenger_cancellation',
+    'provider_cancellation',
+    'service_start',
+    'trip_end'
+  ],
+  reserved: ['reservation_start', 'comms_restored', 'enter_jurisdiction'],
+  non_operational: ['enter_jurisdiction', 'maintenance', 'service_end'],
+  on_trip: ['trip_start', 'enter_jurisdiction', 'comms_restored', 'trip_resume'],
+  elsewhere: ['leave_jurisdiction', 'comms_restored'],
+  unknown: ['comms_lost'],
+  stopped: ['trip_stop', 'reservation_stop']
```

Comment:
Missing `trip_end` and the two cancellation events too, I think.

Reply:
Same as above comments